src/goto-instrument/cover.cpp

```diff
@@ -218,6 +218,299 @@ void collect_operands(const exprt &src, std::vector<exprt> &dest)
 
 /*******************************************************************\
 
+Function: non_ordered_predicate_expansion
+
+  Inputs:
+
+ Outputs:
+
+ Purpose: A non-ordered predicate contains <=, !=, >=
+
+\*******************************************************************/
+
+std::set<exprt> non_ordered_predicate_expansion(const exprt &src)
```

Comment:
I still insist that these should be marked static - yes, the file is way too large and should be chopped up, but that shouldn't be stopping anyone from marking them static now so that later, when splitting up, those people understand what the usage scope is.